webpack.config.babel.js

```diff
@@ -37,12 +37,17 @@ const productionPlugins = [
     // this is actually the property name https://github.com/kimhou/replace-bundle-webpack-plugin/issues/1
     partten: /throw\s+(new\s+)?[a-zA-Z]+Error\s*\(/g,
     replacement: () => 'return;('
-  }])
+  }]),
+
+  new CopyWebpackPlugin([
+    { from: './dropdown-arrow-down.png', to: 'dropdown-arrow-down.png' }
```

Comment:
Your implementation is correct re: copying this to the dist folder for dev testing, and Webpack is notoriously opaque to figure out so kudos!

That being said the issue with providing a separate PNG file is that it's one more thing for users to import and serve in their own application. It's quite a bit of friction.

Two solutions that I feel are better:
- Creating a `src/arrow.js` component that simply returns a very concise blob of inline `<svg>`. The autocomplete is IE9+ only so this is viable;
- In the CSS specify the `background-image` as an inline base64 png.

I think the `<svg>` solution is better because:
- It's much easier to style, you can use `currentColor` to allow users to trivially change the colour for example;
- It's probably smaller, one `<polygon>` element should be enough;
- It scales beautifully regardless of resolutions.

What do you think?